lib/binary_to_decimal.rb

```diff
@@ -4,6 +4,17 @@
 # The least significant bit is at index 7.
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
+
 def binary_to_decimal(binary_array)
-  raise NotImplementedError
+  decimal = []
+  power_count = 7
```

Comment:
Minor issue:  I suggest making the max power count a constant or use the `binary_array.length`.